tests/perf-test/BaseOptionsList.perf-test.tsx

```diff
@@ -0,0 +1,127 @@
+import {fireEvent} from '@testing-library/react-native';
+import type {RenderResult} from '@testing-library/react-native';
+import React, {useState} from 'react';
+import {measurePerformance} from 'reassure';
+import BaseOptionsList from '@components/OptionsList/BaseOptionsList';
+import type {ListItem} from '@components/SelectionList/types';
```

Comment:
@ShridharGoel please update the type from ListItem to OptionData

Reply:
@ShridharGoel Can you address please